### ISSUE TITLE
Mask prompt or mask method

### ISSUE THREAD

**Issue by luoshuiyue** [turandai/gaussian_surfels#32]
Hello, may I ask what is your method of making mask? If it is extracted based on sam model, may I ask what specific prompt can be provided to extract the bread and the plane under the bread?
![1](https://github.com/turandai/gaussian_surfels/assets/25098355/26439f3d-d592-40b8-937c-119ed5edcf61)
![2](https://github.com/turandai/gaussian_surfels/assets/25098355/0337a010-9f2c-4995-b226-ee7474df5d2a)

**Comment by turandai** [comment]
Hi, we make the mask from the depth image and depth range provided in the original blendedMVS dataset.

**Comment by luoshuiyue** [comment]
Thanks a lot! I have another question to ask. Compared with the original Gaussian Splatting method, Gaussian surfels takes up a lot of  GPU memory during training. What are the reasons or possible reasons for the large memory occupation, and can you provide some ideas to solve the large GPU memory occupation.

**Comment by turandai** [comment]
From my perspective, possible reasons are: 1) the normal priors take up additional memotry, 2) gaussian surfels are back-face-culled during optimization and rendering, so it usually need more points than 3dgs to represent a same scene, 3) regulization terms generate addiitional gradient graph that also take up memory. You may try to deal with the these points. Btw, I remember the increase was not very much in my experiments, I am not sure if there is other problems.

**Comment by luoshuiyue** [comment]
Thank you for your reply. I also want to ask, what is the function of idr format pose data?  I found that the pose data input can be colmap format, before-idr-processed format, and after-idr-processed format.  I found that sometimes these three data can be trained, but for different dataset, I sometimes run the code directly without checking, finding the idr data processing script will report errors.  And the train.py script will also report errors on pose data with idr format. 
![1111](https://github.com/turandai/gaussian_surfels/assets/25098355/af32c1ca-7da1-4b4f-9793-49a81d5def25)
![22222](https://github.com/turandai/gaussian_surfels/assets/25098355/fc5a15e5-3839-4c2f-9a33-a7cef2fe011a)
